tckprocess.adoc

```diff
@@ -0,0 +1,422 @@
+# MicroProfile TCK Process 1.0
+
+Specification Teams under the Eclipse Foundation Specification
+Process MUST produce a Technology Compatibility Kit (TCK) that delivers
+on the promise of enabling multiple compatible implementations.
+
+This document defines:
+
+-   Materials a TCK MUST possess to be considered suitable for
+    delivering portability
+
+-   Process for challenging tests and how these challenges are resolved
+
+-   Means of excluding released TCK tests from certification
+    requirements
+
+-   Policy on improving TCK tests for released specifications
+
+-   Process for self-certification
+
+[#_materials_for_a_tck_release]
+## Materials for a TCK Release
+
+Artifacts:
+
+-   Specification contributors MUST produce the team licensed (EPL, Apache, etc.)
+    version for distribution to Maven Central or other open source
+    channels, usable for automated, non-official testing, and
+    implementation. No compatibility claims can be made on the basis of
+    this TCK.
+
+-   Specification contributors MUST produce a final candidate binary that includes the
+    https://www.eclipse.org/legal/tck.php[EFTL] license. The MicroProfile
+    Specification Committee will sign and promote the Specification's TCK binary
+    for distribution via Eclipse infrastructure on final approval. This
+    is the TCK binary usable for self-certification when one desires to
+    make a claim of compatibility.
+
+-   Both TCK binaries MUST contain the following
+
+    -   User guide outlining
+
+        -   Software requirements
+
+        -   Installation and configuration
+
+        -   How to run the tests
+
+        -   Where to file challenges
```

Comment:
At the moment, this info was not in the user guide. I had a quick look at the cdi-tck but did not find this info. Since it will be a dup info for all specs. I suggest to put the challenge process in a separate place instead.

Reply:
The suggestion is to put the source repo such as https://github.com/eclipse/microprofile-config/issues

Reply:
also add a section - where to file certification request?

Reply:
File challenges on the corresponding repo and mark with the label "Challenge", any committer can add the label "Accepted" or "Rejected".